pages/osx/cot.md

```diff
@@ -0,0 +1,24 @@
+# cot
+
+> The Plain-Text Editor for macOS.
+> More information: <https://coteditor.com/>.
+
+- Start CotEditor:
+
+`cot`
+
+- Open a new blank document:
+
+`cot --new`
+
+- Open specific files:
+
+`cot {{path/to/file1 path/to/file2 ...}}`
```

Comment:
```suggestion
- Open specific files in background:

`cot {{path/to/file1 path/to/file2 ...}}`

- Open a new blank document in background:

`cot --new`
```

- See #8018 why I suggested this. 😄
- Is it right that by default everything is opened in background in this editor? If it's true it's better to specify this explicitly assuming that in general editor opens files in foreground by default (until otherwise noted explicitly).

I can't try this editor, so please check my edits carefully.

Reply:
After testing the behavior of running `cot`, it appears that the windows will in fact always open in the background, even though the `-g`/`--background` flag is not provided. This might be a bug with `cot`, although I'm not completely sure.

Because of this, I'd prefer to not specifically state that the command opens file in the background, although I can see why we might want to document it anyway, as it is assumed that `cot` would open files in the foreground by default.

(I will make the change to swap the order of `cot --new` and `cot {{path/to/file1 path/to/file2 ...}}` right now)

Reply:
Also, to me, "Open specific files in **the** background" sounds a bit better than "Open specific files in background". If we keep the "in background" wording (see comment above), would that be a better way to word it, or should I keep it without the "the"?

Reply:
Let's add `the`.

Reply:
Okay, cool. I've made these changes.